app/page/template/video-calling.htm

```diff
@@ -1,110 +1,94 @@
-        <!-- /ko -->
-        <!-- ko if: multitasking.isMinimized() -->
-          <div class="video-element-remote-fullscreen icon-fullscreen hide-controls-hidden"
-               data-uie-name="do-call-controls-video-maximize"
-               data-bind="click: clickedOnMaximize">
-          </div>
+          <div class="video-element-overlay hide-controls-hidden" data-bind="event: {dblclick: doubleClickedOnRemoteVideo}"></div>
         <!-- /ko -->
 
         <!-- ko if: isChoosingScreen() -->
           <choose-screen params="cancel: clickedOnCancelScreen,
-                                 choose: clickedOnChooseScreen,
-                                 screens: availableDevices.screen_input"></choose-screen>
+                                      choose: clickedOnChooseScreen,
+                                      screens: availableDevices.screen_input"></choose-screen>
         <!-- /ko -->
 
         <!-- ko if: showControls() -->
           <div id="video-title" class="video-title"
-               data-bind="css: {'hide-controls-hidden': !showRemoteParticipant() && !isChoosingScreen(),
-                                'video-element-dark': showRemoteParticipant()}">
+            data-bind="css: {'hide-controls-hidden': !showRemoteParticipant() && !isChoosingScreen(), 'video-element-dark': showRemoteParticipant()}">
             <div class="video-remote-name" data-bind="text: videodCall().conversationEntity.display_name()"></div>
             <div class="video-timer label-xs" data-bind="text: z.util.TimeUtil.formatSeconds(videodCall().durationTime())"></div>
           </div>
         <!-- /ko -->
 
         <!-- ko if: showControls() && !isChoosingScreen() -->
-          <div id="video-controls" class="video-controls"
-               data-bind="css: {'hide-controls-hidden': !showRemoteParticipant()}">
-            <div class="video-controls-button button-round button-round-md icon-chat"
-                 data-uie-name="do-call-controls-video-minimize"
-                 data-bind="click: clickedOnMinimize, css: {'button-round-dark': !showRemoteParticipant()}"></div>
-            <div class="video-controls-button button-round button-round-md icon-mute"
-                 data-uie-name="do-call-controls-video-call-mute"
-                 data-bind="click: clickedOnMuteAudio, css: {'toggled': !selfStreamState.audioSend(), 'button-round-dark': !showRemoteParticipant()}"></div>
-            <!-- ko if: showToggleScreen() -->
-              <div class="video-controls-button button-round button-round-md icon-screensharing"
-                   data-bind="click: clickedOnShareScreen, css: {'toggled': selfStreamState.screenSend(), 'button-round-dark': !showRemoteParticipant(), 'disabled': disableToggleScreen()}"
-                   data-uie-name="do-call-controls-toggle-screen"></div>
-            <!-- /ko -->
+          <div id="video-controls" class="video-controls" data-bind="css: {'hide-controls-hidden': !showRemoteParticipant()}">
```

Comment:
Note for later: Not sure this condition is still relevant `{'hide-controls-hidden': !showRemoteParticipant()}`.